### ISSUE TITLE
Mediamanager Item within a RichText field is not working in retrofit mode.

### ISSUE THREAD

**Issue by justsayantan** [sdl/dxa-modules#23]
Mediamanager Item within a RichText field is not working while using retrofit mode. It is always getting failed to map the mediamanager item. 
Error Details - 

```
Sdl.Web.Common.DxaException: Unable to map field 'Content' to property of type 'Sdl.Web.Common.Models.RichText'. ---> Sdl.Web.Common.DxaException: No semantic mapping found between Schema 33274 (uuid:9389a8ce-0f19-44ee-9eb6-2d04d0c02c25:ExternalContentLibraryStubSchemamm) and model type 'Sdl.Web.Common.Models.MediaItem'
   at Sdl.Web.Common.Mapping.SemanticSchema.GetModelTypeFromSemanticMapping(Type baseModelType)
``` 

As a workaround we have created a subclass and extended the default MediaManagerDistribution model. But need proper fix for that.

**Comment by rpannekoek** [comment]
Indeed, the `MediaManagerDistribution` View Model currentky doesn't work properly in conjunction with retrofit mode. The problem is that it only has a `SemanticEntity` annotation for the Core Vocabulary, but in retrofit mode, each CM Schema has its own Vocabulary (named after the Schema's namespace URI). So: an additional `SemanticEntity` annotation is needed for the MM Stub Schema.  Indeed, a work-around for this issue it to create abd register your own subclass which does have the appropriate `SemanticEntity` annotation.

**Comment by rpannekoek** [comment]
Internal Issue ID (for tracking purposes): CRQ-13306
